src/API/Trace/SpanContext.php

```diff
@@ -4,92 +4,16 @@
 
 namespace OpenTelemetry\API\Trace;
 
-use OpenTelemetry\API\Trace as API;
-use function strlen;
-use function strtolower;
-
-final class SpanContext implements API\SpanContextInterface
+final class SpanContext implements SpanContextInterface
 {
     public const INVALID_TRACE = '00000000000000000000000000000000';
-    public const TRACE_VERSION_REGEX = '/^(?!ff)[\da-f]{2}$/';
     public const VALID_TRACE = '/^[0-9a-f]{32}$/';
     public const TRACE_LENGTH = 32;
     public const INVALID_SPAN = '0000000000000000';
     public const VALID_SPAN = '/^[0-9a-f]{16}$/';
-    public const SPAN_LENGTH = 16;
     public const SPAN_LENGTH_BYTES = 8;
```

Comment:
Many of these constants are moved to their respective validator and not used anymore.